activiti-cloud-service-common/activiti-cloud-services-common-identity-keycloak/src/main/java/org/activiti/cloud/services/identity/keycloak/KeycloakManagementService.java

```diff
@@ -242,19 +242,16 @@ public List<SecurityResponseRepresentation> getApplicationPermissions(String app
 
     @Override
     public List<User> findUsersByGroupName(String groupName) {
-        List<User> users = new ArrayList<>();
-        if (!StringUtils.isEmpty(groupName)) {
-            Optional<Group> groupOptional = findGroupStrictlyEqualToGroupName(groupName);
-            groupOptional.ifPresent(group ->
-                    users.addAll(getUsersByGroupId(group.getId())));
-        }
-        return users;
+        if (StringUtils.isEmpty(groupName)) throw new IdentityInvalidGroupException(groupName);
```

Comment:
it works for null, may be you could use spring assertMethods here?

Reply:
With spring Assert I can't throw a specific `IdentityInvalidGroupException` exception